website/content/en/status/report-2023-01-2023-03/bsd-user.adoc

```diff
@@ -7,7 +7,7 @@ link:https://gitlab.com/qemu-project/qemu[QEMU Project's gitlab mirror] URL: lin
 
 Contact: Warner Losh <imp@FreeBSD.org>
 
-In this quarter, Warner upstreamed patche sets in the qemu-project repo (with a third pending).
+In this quarter, Warner upstreamed two patch sets in the qemu-project repo (with a third pending).
```

Comment:
For a third thing to be pending, I assume that the two things preceding were _patch sets_.

True, @bsdimp?